cachecloud-open-web/src/test/java/com/sohu/cache/util/StringUtilTest.java

```diff
@@ -0,0 +1,155 @@
+package com.sohu.cache.util;
+
+import com.sohu.cache.util.StringUtil;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringUtilTest {
+
+  @Test
+  public void convertToCamelCaseStringInputNotNullFalseOutputNotNull() {
+    // Arrange
+    final String inputString = ",";
+    final boolean firstCharacterUppercase = false;
+    // Act
+    final String retval = StringUtil.convertToCamelCaseString(inputString, firstCharacterUppercase);
+    // Assert result
+    Assert.assertEquals(",", retval);
+  }
+
+  @Test
+  public void convertToCamelCaseStringInputNotNullTrueOutputNotNull() {
+    // Arrange
+    final String inputString = "\u1020 ";
+    final boolean firstCharacterUppercase = true;
+    // Act
+    final String retval = StringUtil.convertToCamelCaseString(inputString, firstCharacterUppercase);
+    // Assert result
+    Assert.assertEquals("\u1020", retval);
+  }
+
```

Comment:
The tests above don't sit right with me. What are they are actually doing? Are they just showing a potential bug with their code? Can we encourage Diffblue Cover to give us better inputs?

Reply:
This is the description of the method. In both cases (these two tests), the result is no change to the string as the inputString does not contain any of the characters to be removed. The unicode trace looks a bit odd but it comes out with the right result:
`    /**
     * Description: Remove {_, -, @, $, #, /, &} in string and make letter after
     * this uppercase.<br>
     * e.g. ni_lea-der@gmail./com -> niLeaDerGmail.Com
     * 
     * @param @param inputString
     * @param @param firstCharacterUppercase The first letter is need uppercase.
     * @return String
     * @throws
     */`
It would be good to have a test with inputs similar to the example given in this comment.

Reply:
Changed these two tests to have input strings that will show the effects of the method under test.
All test run and pass.